spec/support/fake_bundler_matchers.rb

```diff
@@ -43,7 +43,7 @@
 
       <<~MSG
         Bundler was given unexpected arguments.
-        Expected #{given} but got #{actual}
+        Expected #{actual} but got #{given}
```

Comment:
I _think_ you're right that these backwards. Nice catch. I still find this a little confusing, however. Would it be clearer to rename `actual` to `expected` and `given` to `actual` throughout to match the RSpec naming convention? Maybe as a separate PR?

Reply:
Thank you, I agree that `expected` and `actual` is much more clear! Since the changes are few, I'm reusing this PR if you don't mind.